extensions/wikia/Search/classes/IndexService/CrossWikiCore.php

```diff
@@ -65,113 +69,130 @@ protected function getWikiBasics() {
 		$response['all_domains_mv_wd'] = $response['domains_txt'];
 
 		$response['wiki_pagetitle_txt'] = str_replace( '$1 - ', '', ( \wfMessage( 'Pagetitle' )->text() ) );
+
 		return $response;
 	}
-	
+
 	/**
 	 * Retrieves the views on a weekly and monthly basis for this wiki
+	 *
 	 * @return array
 	 */
 	protected function getWikiViews() {
 		$result = [];
-		if ( $wvResponse = (new WikiViews)->getStubbedWikiResponse() ) {
-			$result = [ 'views_weekly_i' => $wvResponse['contents']['wikiviews_weekly']['set'], 'views_monthly_i' => $wvResponse['contents']['wikiviews_monthly']['set'] ];
+		if ( $wvResponse = ( new WikiViews )->getStubbedWikiResponse() ) {
+			$result = [
+				'views_weekly_i' => $wvResponse['contents']['wikiviews_weekly']['set'],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "contents" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)